modules/core/react/lib/InputProvider.tsx

```diff
@@ -231,7 +231,7 @@ export default class InputProvider extends React.Component<InputProviderProps> {
     const fn = enable ? window.addEventListener : window.removeEventListener;
 
     // pointer events (mouse, pen, touch)
-    if (window.PointerEvent) {
+    if ('PointerEvent' in window) {
```

Comment:
TS complained that `PointerEvent` is always defined and is a function and that maybe we meant to call it instead.

Reply:
It seems like these changes broke the `InputProvider` tests. Not quite sure why

Reply:
I just changed this back and `yarn typecheck` works and the tests pass?